spec.emu

```diff
@@ -47,24 +27,59 @@ contributors: Jordan Harband
         <emu-alg>
           1. Let _str_ be ? ToString(_S_).
           1. Let _cpList_ be StringToCodePoints(_str_).
-          1. Let _punctuators_ be the following String, which consists of every ASCII punctuator except U+005F (LOW LINE): *"(){}[]|,.?\*+-^$=<>\/#&!%:;@~'"`"*.
-          1. Let _toEscape_ be StringToCodePoints(_punctuators_).
           1. Let _escapedList_ be a new empty List.
           1. For each code point _c_ in _cpList_, do
             1. If _escapedList_ is empty and _c_ is matched by |DecimalDigit|, then
-              1. Append code unit U+005C (REVERSE SOLIDUS) to _escapedList_.
-              1. Append code unit U+0078 (LATIN SMALL LETTER X) to _escapedList_.
-              1. Append code unit U+0033 (DIGIT THREE) to _escapedList_.
-            1. Else if _toEscape_ contains _c_ or _c_ is matched by |WhiteSpace|, then
-              1. Append code unit U+005C (REVERSE SOLIDUS) to _escapedList_.
-            1. Append _c_ to _escapedList_.
+              1. Append code point U+005C (REVERSE SOLIDUS) to _escapedList_.
+              1. Append code point U+0078 (LATIN SMALL LETTER X) to _escapedList_.
+              1. Append code point U+0033 (DIGIT THREE) to _escapedList_.
```

Comment:
```suggestion
              1. Append the code point U+005C (REVERSE SOLIDUS) to _escapedList_.
              1. Append the code point U+0078 (LATIN SMALL LETTER X) to _escapedList_.
              1. Append the code point U+0033 (DIGIT THREE) to _escapedList_.
```

This is still not fixed.

Reply:
whoops, thanks, fixed with the next push.